design/ast.md

```diff
@@ -0,0 +1,14 @@
+# SeedAST
+
+The SeedAST (Seed Abstract Syntax Tree) is a high-level representation of the
+SeedBlock program structure. It can be thought of as an abstract representation
```

Comment:
Not only for SeedBlock, but also for all SeedX languages. SeedAST defines a common subset among SeedBlock and all SeedX languages.

Reply:
Done